datasets/klue/klue.py

```diff
@@ -0,0 +1,521 @@
+            classification benchmark. In this task, given a news headline, a text classifier must
+            predict a topic which is one of politics, economy, society, culture, world, IT/science,
+            and sports. Macro-F1 score is used to evaluate a system."""
+            ),
+            data_url=_DATA_URLs["ynat"],
+            url=_DESCRIPTION_URLs["ynat"],
+            file_map={
+                "train": "ynat-v1_train.json",
+                "dev": "ynat-v1_dev.json",
+            },
+        ),
+        KlueConfig(
+            name="sts",
+            features={
+                "guid": datasets.Value("string"),
+                "source": datasets.Value("string"),
+                "sentence1": datasets.Value("string"),
+                "sentence2": datasets.Value("string"),
+                "labels": {
+                    "label": datasets.Value("float64"),
+                    "real-label": datasets.Value("float64"),
+                    "binary-label": datasets.ClassLabel(names=["negative", "positive"]),
+                },
+            },
+            description=textwrap.dedent(
+                """\
+            STS is a task which aims to predict the semantic similarity of two input sentences as
+            a real value between 0 and 5. Note that we furthure binarized the prediction scores
+            into two classes with a threshold score 3.0 (paraphrased or not) and evaluated with
+            a classification metric.
+            """
+            ),
+            data_url=_DATA_URLs["sts"],
+            url=_DESCRIPTION_URLs["sts"],
+            file_map={
+                "train": "klue-sts-v1_train.json",
+                "dev": "klue-sts-v1_dev.json",
+            },
+        ),
+        KlueConfig(
+            name="nli",
+            features={
+                "guid": datasets.Value("string"),
+                "source": datasets.Value("string"),
+                "premise": datasets.Value("string"),
+                "hypothesis": datasets.Value("string"),
+                "label": datasets.ClassLabel(names=["entailment", "neutral", "contradiction"]),
+            },
+            description=textwrap.dedent(
+                """\
+            NLI is a task to infer the relationship between a hypothesis sentence and a premise
+            sentence. Given the premise, the model determines if the hypothesis is true (entailment),
+            false (contradiction), or undetermined (neutral).
+            """
+            ),
+            data_url=_DATA_URLs["nli"],
+            url=_DESCRIPTION_URLs["nli"],
+            file_map={
+                "train": "klue-nli-v1_train.json",
+                "dev": "klue-nli-v1_dev.json",
+            },
+        ),
+        KlueConfig(
+            name="ner",
+            features={
+                "sentence": datasets.Value("string"),
+                "tokens": datasets.Sequence(datasets.Value("string")),
+                "ner_tags": [
```

Comment:
nitpick: could you use `Sequence` here ? it doesn't change anything accept adding consistency
```suggestion
                "ner_tags": datasets.Sequence(
```
you may need to regenerate dataset_infos.json after this change